charts/memgraph/templates/statefulset.yaml

```diff
@@ -59,14 +59,18 @@ spec:
             - name: bolt
               containerPort: 7687
               protocol: TCP
-          # livenessProbe:
-          #   httpGet:
-          #     path: /
-          #     port: http
-          # readinessProbe:
-          #   httpGet:
-          #     path: /
-          #     port: http
+          readinessProbe:
+            tcpSocket:
+              port: 7687
+            initialDelaySeconds: 5
+            periodSeconds: 10
+            failureThreshold: 3
+          livenessProbe:
+            tcpSocket:
+              port: 7687
+            initialDelaySeconds: 5
+            periodSeconds: 10
+            failureThreshold: 3
```

Comment:
Better with tcpsocket, WDYT ?
tested with Tilt and it's working

Reply:
That for a first version, this will do the job. And I beleive that this is what neo4j uses.